controller_manager/src/controller_manager.cpp

```diff
@@ -2322,6 +2322,10 @@ std::vector<std::string> ControllerManager::get_controller_names()
 
 void ControllerManager::read(const rclcpp::Time & time, const rclcpp::Duration & period)
 {
+  if (periodicity_stats_.GetCount() >= 100)
```

Comment:
the window size could be made configurable

Reply:
good point. Could you please add a param for that? I think 100 is a good default value for resetting

Reply:
I think it is better to leave it to the user to define it. If not defined, let it accumulate from the beginning.